applications/launchpad/gui-react/src/containers/MiningContainer/MiningBox/styles.ts

```diff
@@ -7,3 +7,19 @@ export const MiningBoxContent = styled.div`
   justify-content: space-between;
   flex: 1;
 `
+
+export const NodeIcons = styled.div<{ $color: string }>`
```

Comment:
what's the `$` for ? smells like angular

Reply:
Prop passed to the styled component with `$` prefix makes that it isn't added to the DOM.
(Already described in one of comments in previous PR)

SC complains particularly when we pass boolean. I found one issue in Tabs component, when we are passing `inverted: boolean`. This is what can be found in console or running `yarn test`:

![image](https://user-images.githubusercontent.com/11715931/167145632-632ea346-f04a-4188-96e6-265b05d9fc11.png)

This PR fixes this issue.